pkg/scheduler/objects/queue.go

```diff
@@ -90,6 +90,7 @@ type Queue struct {
 	queueEvents                        *schedEvt.QueueEvents
 	appQueueMapping                    *AppQueueMapping // appID mapping to queues
 	quotaChangePreemptionDelay         uint64
```

Comment:
I just noticed... this change is already in, but worth considering:

1) We already have a variable `preemptionDelay` whose type is `time.Duration`, we might want to have the same for consistency.
2) There is a separate config entry for `Preemption`, but this is also inconsistent with regular preemption settings, which are queue properties. Shouldn't we just go ahead and use properties for both?